.gitignore

```diff
@@ -4,4 +4,5 @@
 target
 work
 .idea
-junit-attachments.iml
\ No newline at end of file
+*.iml
+out/
```

Comment:
good idea

Reply:
thanks!
it is all magic from .ignore plugin https://plugins.jetbrains.com/plugin/7495--ignore